app/api/auth.py

```diff
@@ -220,18 +220,17 @@ def reset_password_post():
     except TypeError:
         return BadRequestError({'source': ''}, 'Bad Request Error').respond()
 
-    try:
+    user_count = get_count(User.query.filter_by(email=email))
+    if user_count:
```

Comment:
Now, it hits DB 2 times instead on one

Reply:
Actually to omit the duplicate messages I had to discard the try-except-else block. If except block will be there then there must be a message too. Any suggestions how can I remove duplication then?

Reply:
>  If except block will be there then there must be a message too

Why?

Reply:
If I get exception NoObjectFound then there should be some code in the block, right? It shouldn't be empty

Reply:
> Log the email

And you can also use `pass`, however, in this case you have to log the email

Reply:
I've used pass for `NoResultFound` case